basilisk/gui/ocr_handler.py

```
@@ -0,0 +1,336 @@
+				+ _("Do you want to open the files?")
+			)
+			if (
+				wx.MessageBox(msg, _("Result"), wx.YES_NO | wx.ICON_INFORMATION)
+				== wx.YES
+			):
+				for file_path in data:
+					wx.LaunchDefaultApplication(file_path)
+		elif isinstance(data, str) and data:
+			wx.MessageBox(data, _("Result"), wx.OK | wx.ICON_INFORMATION)
+		else:
+			log.warning(f"{data}")
```

Pylint (3.3.7)

[warning] Bad indentation. Found 3 spaces, expected 12

(W0311)